ci/test_files_touched.py

```diff
@@ -78,41 +78,37 @@
         'tern -l report -m spdxtagvalue -i photon:3.0',
         'tern -l report -d samples/alpine_python/Dockerfile'],
     # tern/tools
-    re.compile('tern/tools'): 'tern -l report -i golang:alpine',
+    re.compile('tern/tools'): ['tern -l report -i golang:alpine'],
     # tern/utils
     re.compile('tern/utils'): [
         'tern -l report -i golang:alpine',
         'tern -l report -d samples/alpine_python/Dockerfile'],
     # tests
     re.compile('tests/test_class_command.py'):
-        'python tests/test_class_command.py',
+        ['python tests/test_class_command.py'],
     re.compile('tests/test_class_docker_image.py'):
-        'python tests/test_class_docker_image.py',
+        ['python tests/test_class_docker_image.py'],
     re.compile('tests/test_class_image.py'):
-        'python tests/test_class_image.py',
+        ['python tests/test_class_image.py'],
     re.compile('tests/test_class_image_layer.py'):
-        'python tests/test_class_image_layer.py',
+        ['python tests/test_class_image_layer.py'],
     re.compile('tests/test_class_notice.py'):
-        'python tests/test_class_notice.py',
+        ['python tests/test_class_notice.py'],
     re.compile('tests/test_class_notice_origin.py'):
-        'python tests/test_class_notice_origin.py',
+        ['python tests/test_class_notice_origin.py'],
     re.compile('tests/test_class_origins.py'):
-        'python tests/test_class_origins.py',
+        ['python tests/test_class_origins.py'],
     re.compile('tests/test_class_package.py'):
-        'python tests/test_class_package.py',
+        ['python tests/test_class_package.py'],
     re.compile('tests/test_class_template.py'):
-        'python tests/test_class_template.py'
+        ['python tests/test_class_template.py']
         }
 
 alltests = []
 for change in changes:
     for check in test_suite.keys():
         if check.match(change):
-            # add to list differently based on type
-            if isinstance(test_suite[check], list):
                 alltests.extend(test_suite[check])
```

Comment:
I think this line needs to be indented in. Run `pycodestyle ci/test_files_touched.py` to find PEP8 errors.

Reply:
I found a few lines that were too long with pycodestyle and amended my commit to fix that but nothing came up for the indentation:

(ternenv) rjudge@ubuntu:~/ternenv/tern$ pycodestyle ci/test_files_touched.py 
(ternenv) rjudge@ubuntu:~/ternenv/tern$ 

But just looking at it, I noticed the alltests.extend line was indented too far, so fixed that too. Thanks, Nisha!